packages/api/src/db/article.ts

```diff
@@ -49,7 +50,7 @@ export interface Article extends ArticleData {
 }
 
 export interface PeerArticle {
-  peerID: string
+  newsroomID: string
```

Comment:
Since we are making breaking changes, could we actually call this newsroomId instead? Would be amazing :D